src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.ps1

```diff
@@ -19,3 +19,57 @@ foreach ($file in Get-ChildItem $ToolRuntimePath *.runtimeconfig.json)
 # the init-tools.cmd (that is checked into each repository that uses buildtools) can write the semaphore
 # marker into this file once tool initialization is complete.
 New-Item -Force -Type Directory (Join-Path $ToolRuntimePath (Split-Path -Leaf (Split-Path $BuildToolsPackageDir)))
+
+# Download the package version props file, if  was passed in the environment.
+$packageVersionPropsUrl = $env:PB_PACKAGEVERSIONPROPSURL
```

Comment:
Lets not use PB_ variables directly in our build scripts. Just remove the PB_ and it is up to the build definitions to set the correct value in the environment. 

PB_ means it comes from the build definition and should be passed or set to something else we don't want to start depending on them directly outside of our build definitions.

Reply:
Agreed, didn't consider how simple the mapping can be. Just this in `variables`, for a simple impl:

```
"PACKAGEVERSIONPROPSURL": {
  "value": "$(PB_PackageVersionPropsUrl)"
},
```

(I figure all caps is easier to understand later because you don't need to know/realize that VSTS uppercases everything.)